plugins/user-settings/src/components/FeatureFlags/FeatureFlagsItem.tsx

```diff
@@ -31,20 +31,17 @@ type Props = {
 };
 
 export const FlagItem = ({ flag, enabled, toggleHandler }: Props) => (
-  <ListItem>
-    <ListItemText
-      primary={flag.name}
-      secondary={`Registered in ${flag.pluginId} plugin`}
-    />
-    <ListItemSecondaryAction>
-      <Tooltip placement="top" arrow title={enabled ? 'Disable' : 'Enable'}>
-        <Switch
-          color="primary"
-          checked={enabled}
-          onChange={() => toggleHandler(flag.name)}
-          name={flag.name}
-        />
-      </Tooltip>
-    </ListItemSecondaryAction>
-  </ListItem>
+  <>
```

Comment:
Don't think these need to be wrapped in a fragment right? Can just be `<ListItem>` as the root

Reply:
Right, my bad I was using a <Divider> earlier!